### ISSUE TITLE
dependency `python-suds` does not work on MacOSX 

### ISSUE THREAD

**Issue by dilawar** [BhallaLab/moose-gui#4]
I've install suds using pip yet the moosegui does not launch: (On MacOSX).

```
File "/opt/local/lib/moose/gui/biomodelsclient.py", line 49, in <module>
    from suds.client import Client
ImportError: No module named suds.client
```

This is probably related to this http://stackoverflow.com/questions/14973852/suds-install-error-no-module-named-client

/cc @hrani @subhacom @aviralg

**Comment by dilawar** [comment]
`pip install suds-jurko` or (`easy_install suds-jurko`) does not remove the problem. Both packages are also in MacPort. Installing them also does not remove the issue.

**Comment by dilawar** [comment]
Works with `suds-jurko` on MacOSX. Does not work with `suds`. Using `suds-jurko` as dependency now. It is a good idea to 

- Either keep working suds in code-repository as external module.
- Or update the suds on pypi.

**Comment by hrani** [comment]
I removed python-suds from my system which is ubunut 12.04 and installed
suds-jurko, then gui launches without any problem.
so we can removed python-suds and install suds-jurko one package for both
Ubuntu and mac, but need to check for windows.

Harsha

**Comment by subhacom** [comment]
I wrote that code back in 2010 and the original suds module has been discontinued long since: http://stackoverflow.com/questions/7817303/what-soap-libraries-exist-for-python-3-x. In between we had stopped shipping the biomodels client with moosegui. Decide if it is still desired. If so  Jurko's fork `suds-jurko` seems to be the right module (as you have already figured out).

**Comment by dilawar** [comment]
@hrani  Can `biomodels` be dropped from gui ?

**Comment by hrani** [comment]
why do you want to remove biomodel client? when suds-jurko can be installed and works fine atleast on Ubuntu and same can be checked on mac and windows.

**Comment by dilawar** [comment]
Suds-jurko is not available as package on many RPM based distros. Python-suds is
available. If no one is maintaining `bioModels` or it is not used at all, we can
deprecate the folder and reduce one dependencies. 

best,
     Dilawar

**Comment by hrani** [comment]
Out of habit please don't go a head and delete the files.
BioModels is one if database which is active currently So I am sure that they are maintaining it I can check that.

Can you let me know to what all distribution you are trying to packaging.
Fedora, centos, Ubunut, mac (which one) and windows (which one)
I will check if there is a common module which is present which can be used for all the distribution 
For time being if this the time limiting factor for your packaging then I have commented those lines in mgui and biomodel.py file which should not created problems now.
If you still have problems let me know

**Comment by dilawar** [comment]
I've finished the packaging with ghevar_3.0.2. tag. And submitted the files with sha256. We can take this up in next bug-fixes release. On unix, there does not seem to be a problem on recent versions. The list of distribution is [here](https://software.opensuse.org/download.html?project=home%3Amoose&package=moose).

For mac: 10.9, 10.10 and 10.11.  

I am not packaging for windows at all.

**Comment by dilawar** [comment]
Using `python-suds-jurko` on most of the distributions.

**Comment by dilawar** [comment]
Not available on CentOS-6 .

**Comment by dilawar** [comment]
Added suds to source code in pull request #9

**Comment by dilawar** [comment]
This has been fixed by recent merge of pull request #9
